webui/react/src/pages/Templates/TemplatesList.tsx

```diff
@@ -169,8 +177,19 @@ const TemplateList: React.FC<Props> = ({ workspaceId }) => {
   );
 
   const columns = useMemo(() => {
+    const actionMenu = (record: Template) => {
+      const menu: MenuItem[] = [{ key: MenuKey.ViewTemplate, label: 'View Template' }];
```

Comment:
**question:** just confirming that `View Template` is always available and is not RBAC dependent?

Reply:
If a user does not have permission to view the template, then that template should not show up in the table at all.